eslint.config.mjs

```diff
@@ -37,9 +38,14 @@ invariant(react.configs.flat, 'For typescript');
 invariant(react.configs.flat.recommended, 'For typescript');
 invariant(react.configs.flat['jsx-runtime'], 'For typescript');
 
+// Exclude MDX files from type-aware linting
+// https://github.com/orgs/mdx-js/discussions/2454
+const MDXIgnore = ['**/*.mdx'];
```

Comment:
looks like this is also used in `plugin/typescript-eslint/custom` which should not include type-aware rules

Reply:
I need to update the comment, it should say typescript linting.